src/commands/project/delete/source.ts

```diff
@@ -71,12 +72,16 @@ export class Source extends SfCommand<DeleteSourceJson> {
       description: messages.getMessage('flags.wait.description'),
       summary: messages.getMessage('flags.wait.summary'),
     }),
+    tests: {
+      ...testsFlag,
+      helpGroup: testFlags,
+      char: undefined,
+    },
     'test-level': testLevelFlag({
       aliases: ['testlevel'],
       deprecateAliases: true,
       description: messages.getMessage('flags.test-Level.description'),
       summary: messages.getMessage('flags.test-Level.summary'),
-      options: ['NoTestRun', 'RunLocalTests', 'RunAllTestsInOrg'],
```

Comment:
use the defaults set in the flag:
https://github.com/salesforcecli/plugin-deploy-retrieve/blob/cd/delete-support-specific-tests/src/utils/flags.ts#L74